jekyll/_cci2/server/v4.1/overview/release-notes.adoc

```diff
@@ -33,6 +33,21 @@ Find information about migration in the following guides:
 == Upgrade
 For upgrade steps see the xref:../installation/upgrade-server#[Upgrade server v4 guide].
 
+[#release-4-1-3]
+== Release 4.1.3
+
+[#changelog-4-1-3]
+=== Changelog
+
+For full details of this release see the link:https://circleci.com/server/changelog/#release-4-1-3[changelog].
```

Comment:
This has been merged but I opened another PR to change the date to the actual release date (i.e. 25th May instead of 24th).

Reply:
Thanks, just approved the changelog